packages/relay/src/lib/eth.ts

```diff
@@ -1646,6 +1678,7 @@ export class EthImpl implements Eth {
     }
 
     await this.batchGetAndPopulateContractResults(contractResults, showDetails, transactionObjects, transactionHashes, requestIdPrefix);
+    await this.filterAndPopulateSyntheticContractResults(showDetails, logs, transactionObjects, transactionHashes, requestIdPrefix);
```

Comment:
Let's feature gate this if we're cherry picking.
Default to false though

Reply:
should the feature gate be only on the `0.28` branch or should I include it in main too?

Also, do you think we should cherry pick it for `0.28` or is it better to wait for `0.29` to include it?

Reply:
✅  Done